### PR TITLE
[chip/dv] Fix the unmapped hmac idle test

### PR DESCRIPTION
**Pull request by engdoreis**
Signed-off-by: Douglas Reis <doreis@lowrisc.org>

### CODE REVIEW

**Review by weicaiyang — APPROVED** [review]
LGTM

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 1]
```diff
@@ -1778,7 +1778,7 @@
               digest.
             '''
       milestone: V2
-      tests: ["chip_sw_hmac_idle"]
+      tests: ["chip_sw_hmac_enc_idle"]
```
Comment (msfschaffner) [inline hw/top_earlgrey/data/chip_testplan.hjson:1781]: There is still another instance of `chip_sw_hmac_idle` in this testplan. Is that supposed to be changed to `chip_sw_hmac_enc_idle`, too?

Comment (engdoreis) [reply hw/top_earlgrey/data/chip_testplan.hjson:1781]: Yes, sorry I didn't see it.

Comment (msfschaffner) [reply hw/top_earlgrey/data/chip_testplan.hjson:1781]: Ah sorry I believe I have been looking at an outdated nightlies dashboard - nevermind!

**Pull request merged**